script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -715,7 +767,9 @@ That dream was the memory of the Roa before me.
 [sha:3913a280d5058ab61b18419315d0e0edf6b83a97]{
 -- Page 37, Offset 33208.
 -- 部屋に閉じこもって、そのあげくに両親を殺して、自分の住んでいた街を吸血鬼だらけにしてしまった、見知らぬ誰かの、最後の記憶。
-The final memories of an unfamiliar someone――someone who secluded themselves in their room, before eventually killing their parents, and then transforming their hometown into a city rife with vampires.
+The final memories of an unfamiliar someone――
+someone who walled themselves up in their room,
+ but in the end still killed their parents and transformed their hometown into a city infested with vampires.
```

Comment:
"an unfamiliar someone" sounds a bit too awkward when you continue it up here.
These could be structured in a more staggered manners. Could use periods instead of dashes  if you feel them too bothersome.
The last sentence could be a bit more dramatic, and use some slight alternate constructions.
>The final memories of someone I don't know――
>someone who walled themselves up in their room――
>someone who eventually ended up killing their parents and transforming their hometown into a town infested with vampires.